plugins/postgres/dbt/include/postgres/macros/adapters.sql

```diff
@@ -155,6 +155,6 @@
   {% for column_name in column_dict %}
     {% set comment = column_dict[column_name]['description'] %}
     {% set escaped_comment = postgres_escape_comment(comment) %}
-    comment on column {{ relation }}.{{ column_name }} is {{ escaped_comment }};
+    comment on column {{ relation }}.{{ adapter.quote(column_name) if column_dict[column_name]['quote'] == True else column_name }} is {{ escaped_comment }};
```

Comment:
```suggestion
    comment on column {{ relation }}.{{ adapter.quote(column_name) if column_dict[column_name]['quote'] else column_name }} is {{ escaped_comment }};
```

`True` is the only truthy value we need to care about for `quote`, so no need to actually compare it to True.